static/js/pages/instances/CreateInstanceForm.tsx

```diff
@@ -59,85 +75,103 @@ const CreateInstanceForm: FC = () => {
     instanceType: Yup.string().required("Instance type is required"),
   });
 
-  interface FormValues {
-    name: string;
-    image: RemoteImage | undefined;
-    instanceType: string;
-    profiles: string[];
-    yaml?: string;
-    unchangedYaml?: string;
+  function notifyInProgress() {
+    instanceListNotify?.info(
+      <>A new instance has successfully been requested.</>,
```

Comment:
I think it should be `A new instance has been successfully requested.` or just `A new instance has been requested.`

Reply:
Simplified the message.